.github/workflows/compilemmd.yml

```
@@ -0,0 +1,42 @@
+name: Node.js CI
+
+on:
+  push:
```

[want] PR でビルドするようにしますか
```suggestion
  pull_request:
    types: [ opened, reopened, synchronize ]
```